component.json

```diff
@@ -12,6 +12,11 @@
                     "label": "WebHook URL",
                     "required": false
                 },
+                "hmac_key": {
+                    "viewClass": "OptionalRandomString",
```

Comment:
Well, its a good idea, but I'm not sure if it is necessary for the first iteration.
@drobiazko am I right?